clients/js/src/global.d.ts

```diff
@@ -0,0 +1,8 @@
+// Done to appease issue in generated code:
+//  error TS4111: Property 'NODE_ENV' comes from an index signature, so it must be accessed with ['NODE_ENV'].
+
+declare namespace NodeJS {
+  interface ProcessEnv {
+    NODE_ENV: string;
+  }
+}
```

Comment:
Interesting, I wonder if it'd be safer for Codama to generate as `['NODE_ENV']` then.

I added an issue on Codama (https://github.com/codama-idl/codama/issues/523) to discuss this.